### DISCUSSION TITLE
Call Tiled Diffusion & VAE for upscale

### DISCUSSION THREAD

**Discussion by player99963** [AIrjen/OneButtonPrompt#148] [Ideas]
First of all this is a great extension, thank you for making it. It has helped me so much.

It is a great idea to be able to call Ultimate SD Upscale for one-click batch upscale. On top of that, I would like to see this feature further enhanced.
While Ultimate SD Upscale is a great extension, it still has its advantages and disadvantages compared to Tiled Diffusion & VAE.
For me and some other users, we are more accustomed to using Tiled Diffusion & VAE. And in my experience, it handles some details better than Tiled Diffusion & VAE.
Therefore, I think including Tiled Diffusion & VAE as one of the upscale options is a very good enhancement to OneButtonPrompt.

Tiled Diffusion & VAE: https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111

**Comment by AIrjen** [comment]
Hi! Thanks for the suggestion. I also have a suggestion for trying to implement Adetailer.

The problem with adding too much extensions, is trying to keep the interface manageble, and calling these extensions. I am not sure how to call these from within the API.

I will keep this in the back of my mind, when I have some more time to tinker with some things.

**Reply by player99963** [reply]
Thank you for your interest and I look forward to the day when this proposal is realized.
Because honestly, after comparing, I think Tiled Diffusion & VAE can handle details much better than Ultimate SD Upscale.


Regarding the manageability of the interface.
I think the first thing that could be done is to reduce the amount of explanatory text in the interface.
This could be replaced with a small number of links to the documentation, or a small window that appears when the mouse is moved over an option to provide a brief explanation.

I think the user should read the instructions elsewhere, and putting too much text in the interface is detrimental to the user experience, which is putting the cart before the horse.
Even if we don't consider adding more functions, reducing the amount of text will not only make the interface cleaner, but will also improve the user experience.

**Reply by AIrjen** [reply]
Hi, I did some work on the texts. These are now put under a clickable help button.

I also added icons everywhere, but that is more for the non-english speaking audience.

**Reply by player99963** [reply]
Great changes!

I am actually a non-English speaking user. I basically used the translation software to read the explanatory text to familiarize myself with all the operations.

Although it is difficult to understand what the icons mean to someone who is not familiar with the functions, once you do understand them, you can easily remember what they mean when you look at them.
